packages/env/lib/env.js

```diff
@@ -390,6 +390,14 @@ async function configureWordPress( environment, config ) {
 			options
 		);
 	}
+
+	for ( const [ key, value ] of Object.entries( config.config ) ) {
+		await dockerCompose.run(
+			'cli',
+			`wp config set ${ key } ${ value } --raw`,
```

Comment:
From memory, `dockerCompose` is pretty naive and does a `.split( ' ' )` on the string we pass here. This will cause problems if `value` has a space in it. You can get around it by passing an array instead e.g. `[ 'wp', 'config', 'set', key, value, '--raw' ]`.

Reply:
Good catch.